machine_learning/lstm/lstm_prediction.py

```diff
@@ -6,6 +6,7 @@
 """
 import numpy as np
 import pandas as pd
+from tensorflow import keras
```

Comment:
Is this line being used anywhere?

Reply:
@onlinejudge95  Yes ofc  I think it will need
won't it give error of tensorflow??
Coz as keras is part of tensorflow soo...

Reply:
> @onlinejudge95 Yes ofc I think it will need
> won't it give error of tensorflow??
Why should it, `terraform` is not a dependency of this script
> Coz as keras is part of tensorflow soo...
Did you check the [requirements.txt](https://github.com/Jenil-S/Python/blob/my-small-contribution/requirements.txt#L3) file, `keras` is installed as a separate dependency and I see direct imports from `keras` which if some version conflicts between the terraform's provided API and the kera's provided API happen might break things.

I suggest you use either one of them

Reply:
Ohh yeah I just see that so i think i should remove that 
Right?

Reply:
i would suggest go with what the official docs on this one

Reply:
Okay